js/widgets/forms/slider.js

```diff
@@ -409,40 +322,31 @@ return $.widget( "mobile.slider", $.extend( {
 			themeClass = theme ? " ui-button-" + theme : "",
 			trackTheme = this.options.trackTheme || parentTheme,
 			trackThemeClass = trackTheme ? " ui-bar-" + trackTheme : " ui-bar-inherit",
-			cornerClass = this.options.corners ? " ui-corner-all" : "",
-			miniClass = this.options.mini ? " ui-mini" : "",
 			left, width, data, tol,
 			pxStep, percent,
-			control, isInput, optionElements, min, max, step,
+			control, min, max, step,
 			newval, valModStep, alignValue, percentPerStep,
 			handlePercent, aPercent, bPercent,
 			valueChanged;
 
-		self.slider[ 0 ].className = [ this.isToggleSwitch ? "ui-slider ui-slider-switch ui-slider-track ui-shadow-inset" : "ui-slider-track ui-shadow-inset", trackThemeClass, cornerClass, miniClass ].join( "" );
+		this._addClass( $( self.slider[ 0 ] ), "ui-slider-track", trackThemeClass );
```

Comment:
cant this just be self.slider seems like your accessing the element then re creating a new jQuery object for it?